### PR TITLE
Express get

### PR DESCRIPTION
**Pull request by benihime1458**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -11,6 +12,17 @@ app.use((req, res, next) => {
 // serve static assets
 app.use(express.static(path.join(__dirname, '../client/dist')));
 
+// outputs all comments for a song
+app.get('/api/songs/songId/comments', (req, res) => {
```
Comment (kristie-zheng) [inline server/server.js]: Are you trying to have a dynamic songId? If so, use
:songId

File: server/server.js [hunk 2]
```diff
@@ -11,6 +12,17 @@ app.use((req, res, next) => {
...
+  mongoDB.getComments(null, (err, data) => {
```
Comment (kristie-zheng) [inline server/server.js]: what is the first argument, null, for?

File: db/db.js [hunk 3]
```diff
@@ -39,12 +37,25 @@ for (let i = 0; i < sampleData.length; i += 1) {
   });
 }
 
-// console.log(sampleData); 
-
-Comment.find((error, comments) => {
-  if (error) {
-    console.log('could not find comments');
+// Gets all comments for a song on song page
+const getComments = (err, retrievedComments) => {
```
Comment (kristie-zheng) [inline db/db.js]: Where is the specified song passed in?

Comment (benihime1458) [reply db/db.js]: I didn't generate song data, only the comments that will be on the song page.

File: server/server.js (hunk 1 above)
Comment (benihime1458) [reply server/server.js]: It's static and a stand in since I don't handle song ID's.

File: server/server.js (hunk 2 above)
Comment (benihime1458) [reply server/server.js]: The first argument would be an error for getComments function in db.js. Specifying null as the first argument will retrieve the comments instead.

**Pull request merged**